addons/web_editor/static/lib/odoo-editor/test/utils.js

```diff
@@ -274,6 +274,12 @@ export async function testEditor(Editor = OdooEditor, spec, options = {}) {
     const testNode = document.createElement('div');
     document.querySelector('#editor-test-container').innerHTML = '';
     document.querySelector('#editor-test-container').appendChild(testNode);
+    let styleTag;
+    if (spec.styleContent) {
```

Comment:
I like this feature. I'd call it `style` though, like the `<style>` tag that it is actually emulating.